src/clients/shiplift.rs

```diff
@@ -252,20 +258,6 @@ mod tests {
         let image = HelloWorld::default();
         let shiplift = Shiplift::new();
 
-        let mut rt = tokio::runtime::Runtime::new().unwrap();
-        rt.block_on(async {
-            shiplift.run(image).await
-        });
-    }
-
-    #[test]
-    fn shiplift_run_command_should_include_name() {
-        let image = HelloWorld::default();
-        let run_args = RunArgs::default().with_name("hello_container");
-
-        let shiplift = Shiplift::new();
-        shiplift.run_with_args(image, run_args);
-
-        //TODO add assert maybe receive the container and then inspect the id
+        tokio_test::block_on(shiplift.run(image));
```

Comment:
If you depend on `tokio` with the `macros` feature enabled, you can write `#[tokio::test]` instead of `#[test]` and make the test `fn` itself `async` so you can just use `.await` :)

Reply:
Yup! I was looking for that syntax, and didn't dig deeper. 

Btw, do you think this will introduce some inconvenience to the users of the testcontainer crate? To use Async Docker trait you'll have to depend on tokio crate.

Reply:
Yes, it might create some inconvenience which is why I was thinking of keeping both APIs :)